src/java-runtime/index.ts

```diff
@@ -77,3 +78,21 @@ export async function validateJavaRuntime() {
 
   return false;
 }
+
+export async function suggestOpenJdk(jdkVersion: string = "openjdk8", impl: string = "hotspot") {
+  let os: string = process.platform;
+  if (os === "win32") {
+    os = "windows";
+  } else if (os === "darwin") {
+    os = "mac";
+  } else {
+    os = "linux";
+  }
+
+  let arch = process.arch;
```

Comment:
Be careful of using `process.arch`: https://nodejs.org/api/process.html#process_process_arch

> The process.arch property returns a string identifying the operating system CPU architecture for which the Node.js binary was compiled.

So it sometimes will not return the actual system arch. For example, running 32-bit VS Code on an 64-bit OS. See:
<img width="703" alt="Screen Shot 2019-03-30 at 11 24 33 AM" src="https://user-images.githubusercontent.com/6193897/55270805-b5edf900-52de-11e9-96e9-f30844d6f991.png">

Well, it might be a big problem though. If we want to solve this problem, seems that we can leverage this node module: https://www.npmjs.com/package/arch

Reply:
Nice! Thank you!